client/app/templates/projects.hbs

```diff
@@ -6,38 +6,43 @@
   <div class="cell large-8">
 
     {{#if (gt this.model.length 0)}}
-      {{#if (gt this.applicantProjects.length 0)}}
-        <ul class="no-bullet">
-          {{#each this.sortedApplicantProjects as |project|}}
-            <Projects::ApplicantProjectCard @project={{project}} />
+
+      <h2>Projects Awaiting Submission</h2>
+
+      {{#if (gt this.toDoProjects.length 0)}}
+        <ul class="no-bullet" data-test-to-do-projects-list>
```

Comment:
For the most part we've been naming our test selectors so that the specific variant or modifier on a selector is assigned as the selector value. It's somewhat similar to the CSS concept of [BEM](http://getbem.com/introduction/)

i.e. `data-test-<element>="<modifier>"` 

So in this case, instead of `data-test-to-do-projects-list` and `data-test-done-projects-list`, it should be:
`data-test-projects-list="to-do"` and `data-test-projects-list="done"`

Reply:
Updated.

Reply:
@andycochran  hmm I don't see the changes yet.. maybe it didn't push?